app/views/layouts/_user_menu.html.haml

```diff
@@ -21,7 +21,10 @@
     = link_to(destroy_user_session_path, :method=>'delete') do
       %span.fa.fa-minus
       Sign out
-- if can? :manage, Conference
+- if current_user.is_admin || (current_user.has_any_role? :admin, { :name => :organizer, :resource => :any },
```

Comment:
Very nice! Those users, with these roles, must be able to see the link 'Administration' which redirects to /admin/conference. And why they able to see the link? What ability do they have that tells us that they should be seeing and accessing the link 'Administration' ?